tests/attacks/test_wasserstein.py

```diff
@@ -0,0 +1,434 @@
+# MIT License
```

Comment:
Could you please change this test into the new pytest format and move this file to `test.attacks.evasion`?
I think these test have not been running on Travis so far because this file had not been added to `run_tests.sh`.

Reply:
As discussed, we will keep the tests as now and will refactor the tests into the pytest framework when all frameworks are tested. I have also added the test into run_tests.sh.